app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1> RECIPES </h1>
+
+<p> Name: <%= @recipe.name %> </p>
+<p> Complexity: <%= @recipe.complexity %> </p>
+<p> Genre: <%= @recipe.genre %> </p>
+<p> Ingredients: </p>
+<% @recipe.ingredients.each do |ingredient| %>
+  <ul> <%= ingredient.name %>: <%= ingredient.cost %> </ul>
+<% end %>
+<p> Total Cost: $<%= @recipe.total_cost%> </p>
```

Comment:
Great attempt at using `<ul>` although the usage here isn't quite correct. You want to have your `<ul>` on the outside of your `.each` and then each iteration with be an `<li>`.

```erb
<ul>
  <% @recipe.ingredients.each do |ingredient| %>
    <li> <%= ingredient.name %>: <%= ingredient.cost %> </li>
  <% end %>
</ul>
```

Great job abstracting the `total_cost` method to the Model over doing the AR in the view which would be an MVC infraction.